packages/lib-classifier/src/plugins/tasks/SimpleDropdownTask/models/SimpleDropdownTask.spec.js

```diff
@@ -0,0 +1,102 @@
+import { types } from 'mobx-state-tree'
+import SimpleDropdownTask from '@plugins/tasks/SimpleDropdownTask'
+
+const simpleDropdownTask = {
+  instruction: 'Choose your favourite colour',
+  allowCreate: false,
+  options: [
+    'Red',
+    'Blue',
+    'Yellow',
+    'Green',
+    'White',
+    'Black',
+  ],
+  required: false,
+  taskKey: 'T1',
+  type: 'dropdown-simple'
+}
+
+describe('Model > SimpleDropdownTask', function () {
+  it('should exist', function () {
+    const simpleDropdownTaskInstance = SimpleDropdownTask.TaskModel.create(simpleDropdownTask)
+    expect(simpleDropdownTaskInstance).to.be.ok()
+    expect(simpleDropdownTaskInstance).to.be.an('object')
+  })
+
+  it('should error for invalid tasks', function () {
+    let errorThrown = false
+    try {
+      SimpleDropdownTask.create({})
+    } catch (e) {
+      errorThrown = true
+    }
+    expect(errorThrown).to.be.true()
+  })
+
+  describe('with an annotation', function () {
+    let task
+
+    before(function () {
+      task = SimpleDropdownTask.TaskModel.create(simpleDropdownTask)
+      const annotation = task.defaultAnnotation
+      const store = types.model('MockStore', {
+        annotation: SimpleDropdownTask.AnnotationModel,
+        task: SimpleDropdownTask.TaskModel
+      })
+      .create({
+        annotation,
+        task
+      })
+      task.setAnnotation(annotation)
+    })
+
+    it('should start up with a null value', function () {
+      expect(task.annotation.value).to.be.null()
+    })
+
+    it('should update annotations', function () {
+      task.updateAnnotation({
+        value: 'Red',
+        option: true,
+      })
+      const annotationValue = task.annotation.value
+      expect(annotationValue.value).to.equal('Red')
```

Comment:
I think there's an inconsistency with the implementation and the readme. This should be the index of the value, not the string value itself? This is so we can support the workflow task translations I think.

Reply:
The 'dual language dropdown' workflow here has an example where you can try out the dropdown in English and Spanish, and look at the resulting classification. Like @srallen says, the annotation should be the same in either language. https://master.pfe-preview.zooniverse.org/projects/eatyourgreens/-project-testing-ground